src/annotator/components/ContentInfoBanner.js

```diff
@@ -0,0 +1,43 @@
+import { LabeledButton, Link } from '@hypothesis/frontend-shared';
+
+/**
+ * @typedef {import('../../types/annotator').ContentInfoConfig} ContentInfoConfig
+ */
+
+/**
+ * A banner that displays information about the current document and the entity
+ * that is providing access to it (eg. JSTOR).
+ *
+ * @param {object} props
+ *   @param {ContentInfoConfig} props.info
+ *   @param {() => void} props.onClose
```

Comment:
I can handle this on design implementation, but the current proposed design is non-close-able.

Reply:
OK, I'll let you remove this as part of the design update.